prow/plugins/trigger/pr.go

```diff
@@ -128,6 +136,10 @@ func trustedPullRequest(ghc githubClient, pr github.PullRequest) (bool, error) {
 }
 
 func buildAll(c client, pr github.PullRequest) error {
+
+	if err := c.GitHubClient.RemoveLabel(pr.Base.Repo.Owner.Login, pr.Base.Repo.Name, pr.Number, needsOkToTest); err != nil {
```

Comment:
You receive a 404 and 
```
{
  "message": "Label does not exist",
  "documentation_url": "https://developer.github.com/v3/issues/labels/#remove-a-label-from-an-issue"
}
```

Since it's not really a problem if we try to remove the label and it's not there, is there someway to ignore the 404 errors.